### PR TITLE
[2.4] update rancher-k3s-upgrader chart version 0.2.1

### PR DESCRIPTION
**Pull request by MonzElmasry**
This pr updates the version of rancher-k3s-upgrader chart
New version includes a keep policy for the cattle-system namespace so it doest get deleted when chart is uninstalled

### CODE REVIEW

**Review by cjellick — APPROVED** [review]

**Review by rmweir — APPROVED** [review]
LGTM

**Comment by davidnuzik** [comment]
Per Maggie we should not merge this until we have the OK from her.

**Comment by davidnuzik** [comment]
"The 2.5. and 2.4 releases will not be merging charts or system-charts to their respective release branches, so we are still free to merge to those dev branches." So we will merge :)

**Pull request merged**
